exercises/nucleotide-count/canonical-data.json

```diff
@@ -1,12 +1,12 @@
 {
   "exercise": "nucleotide-count",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "cases": [
     {
-      "description": "count all nucleotides in a strand",
+      "description": "nucleotideCounts counts all DNA-nucleotides in the input strand and returns an object with properties A, C, G and T holding their respective amounts.",
```

Comment:
This does not need to be the description of the problem, that is provided in a separate file.

Several tracks generate the test name based on the test descriptions in this file so having a concise description is a good thing.

Reply:
I see, thanks. I'll change this.